### PR TITLE
added chatbot

### LINKED ISSUE #79: Add chatbot for public demoshop
There should be a chatbot everywhere. Please place code below into head everywhere (all domains + admin):

<script type="text/javascript">
    window._chatlio = window._chatlio||[];
    !function(){ var t=document.getElementById("chatlio-widget-embed");if(t&&window.ChatlioReact&&_chatlio.init)return void _chatlio.init(t,ChatlioReact);for(var e=function(t){return function(){_chatlio.push([t].concat(arguments)) }},i=["configure","identify","track","show","hide","isShown","isOnline", "page", "open", "showOrHide"],a=0;a<i.length;a++)_chatlio[i[a]]||(_chatlio[i[a]]=e(i[a]));var n=document.createElement("script"),c=document.getElementsByTagName("script")[0];n.id="chatlio-widget-embed",n.src="https://w.chatlio.com/w.chatlio-widget.js",n.async=!0,n.setAttribute("data-embed-version","2.3");
       n.setAttribute('data-widget-id','824874cb-7e8b-4a4e-5380-10c1697b3d08');
       c.parentNode.insertBefore(n,c);
    }();
</script>

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Easier way to contact Shopsys directly from demoshop
|New feature| Yes
|Fixes issues| closes #79 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Review by vitek-rostislav — CHANGES_REQUESTED** [review]
Hi @TomasLudvik, the chatbot should be displayed also in the administration so you will probably need to overwrite the base admin template as you did in https://github.com/shopsys/demoshop/pull/81

**Comment by TomasLudvik** [comment]
> Hi @TomasLudvik, the chatbot should be displayed also in the administration so you will probably need to overwrite the base admin template as you did in #81

Fixed

File: app/Resources/ShopsysFrameworkBundle/views/Admin/Layout/base.html.twig [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    </head>
+
+    <body>
+        {% block html_body %}{% endblock %}
+
+        {% if app.environment == 'dev' %}
+            <script async src="//localhost:35729/livereload.js"></script>
+        {% endif %}
+
+        <style type="text/css">
```
Comment (vitek-rostislav) [inline app/Resources/ShopsysFrameworkBundle/views/Admin/Layout/base.html.twig:81]: is this because of the fixed bar we have on many pages in the admin?

**Review by vitek-rostislav — APPROVED** [review]
Hi, now it seems ok, I have only one question :wink:

File: app/Resources/ShopsysFrameworkBundle/views/Admin/Layout/base.html.twig (hunk 1 above)
Comment (TomasLudvik) [reply app/Resources/ShopsysFrameworkBundle/views/Admin/Layout/base.html.twig:81]: Exactly

**Review by LukasHeinz — APPROVED** [review]

**Comment by PetrHeinz** [comment]
What should we do about the failing SonarCloud checks introduced in #72? After a short review the recommendations do seem relevant and useful. 

@grossmannmartin: Is the failing check a merge-stopper?

**Pull request merged**